rbi/core/hash.rbi

```diff
@@ -917,23 +917,36 @@ class Hash < Object
   end
   def merge(*arg0, &blk); end
 
-  # Adds the contents of *other_hash* to *hsh*. If no block is specified,
-  # entries with duplicate keys are overwritten with the values from
-  # *other_hash*, otherwise the value of each duplicate key is determined by
-  # calling the block with the key, its value in *hsh* and its value in
-  # *other_hash*.
+  # Adds the contents of the given hashes to the receiver.
   #
-  # ```ruby
+  # If no block is given, entries with duplicate keys are overwritten with the
+  # values from each `other_hash` successively, otherwise the value for each
+  # duplicate key is determined by calling the block with the key, its value in
+  # the receiver and its value in each `other_hash`.
+  #
+  # ```
   # h1 = { "a" => 100, "b" => 200 }
-  # h2 = { "b" => 254, "c" => 300 }
-  # h1.merge!(h2)   #=> {"a"=>100, "b"=>254, "c"=>300}
-  # h1              #=> {"a"=>100, "b"=>254, "c"=>300}
+  # h1.merge!          #=> {"a"=>100, "b"=>200}
+  # h1                 #=> {"a"=>100, "b"=>200}
   #
   # h1 = { "a" => 100, "b" => 200 }
-  # h2 = { "b" => 254, "c" => 300 }
-  # h1.merge!(h2) { |key, v1, v2| v1 }
-  #                 #=> {"a"=>100, "b"=>200, "c"=>300}
-  # h1              #=> {"a"=>100, "b"=>200, "c"=>300}
+  # h2 = { "b" => 246, "c" => 300 }
+  # h1.merge!(h2)      #=> {"a"=>100, "b"=>246, "c"=>300}
+  # h1                 #=> {"a"=>100, "b"=>246, "c"=>300}
+  #
+  # h1 = { "a" => 100, "b" => 200 }
+  # h2 = { "b" => 246, "c" => 300 }
+  # h3 = { "b" => 357, "d" => 400 }
+  # h1.merge!(h2, h3)
+  #                    #=> {"a"=>100, "b"=>357, "c"=>300, "d"=>400}
+  # h1                 #=> {"a"=>100, "b"=>357, "c"=>300, "d"=>400}
+  #
+  # h1 = { "a" => 100, "b" => 200 }
+  # h2 = { "b" => 246, "c" => 300 }
+  # h3 = { "b" => 357, "d" => 400 }
+  # h1.merge!(h2, h3) {|key, v1, v2| v1 }
+  #                    #=> {"a"=>100, "b"=>200, "c"=>300, "d"=>400}
+  # h1                 #=> {"a"=>100, "b"=>200, "c"=>300, "d"=>400}
   # ```
```

Comment:
I think there is one more line:

```suggestion
  # ```
  # [#update](https://docs.ruby-lang.org/en/2.6.0/Hash.html#method-i-update) is an alias for [#merge!](https://docs.ruby-lang.org/en/2.6.0/Hash.html#method-i-merge-21).
```

Reply:
Thanks! I did not know about `rbi/tools/sync-rdoc.rb`, reran it and updated the PR with its output.